src/cli/commands/daemon.js

```diff
@@ -44,8 +44,8 @@ module.exports = {
       const repoPath = getRepoPath()
 
       // Required inline to reduce startup time
-      const HttpApi = require('../../http')
-      const api = new HttpApi({
+      const StandaloneDaemon = require('../../cli/standalone-daemon')
```

Comment:
Can we just call this `Daemon`?

Reply:
Done in https://github.com/ipfs/js-ipfs/pull/1950/commits/a2a307d3cf2887ea5353585e5ac1b8a17eef201f